florist/app/client_imports.tsx

```diff
@@ -0,0 +1,15 @@
+"use client"
+
+import { useEffect } from "react";
+
+function ClientImports() {
```

Comment:
nit: If we are using typescript should we add null return type

Reply:
we need linting for typescript! It's my first time using it so I don't know what I'm doing most of the time  😬

Reply:
I am in the same boat haha we can def find a linter to make our lives easier!